practices/managing-node-projects.md

```diff
@@ -34,23 +36,25 @@ It's important to specify which versions of node your application expects. There
   Your project should include an [`.nvmrc` file](https://github.com/creationix/nvm#nvmrc) in the root directory of the project to specify which version(s) of node are compatible.
   You can then run `nvm use` to use the right version of node. Additionally [Travis respects `.nvmrc` files](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc), so using one will simplify your Travis configuration.
 
-### Run `nvm use` before `npm install`
+### Installing packages predictably
 
-You should always ensure you're using the correct version of node (and implicitly `npm`) before doing an `npm install`.
+To reduce "but it works on my machine" issues, use a method that makes installs more predictable, and minimises changes to the `package-lock.json` (if your project is committing that file - this will be the case if your software is a node _application_, but not if it's a _library_).
 
-Running `nvm use` before `npm install` is good because:
-1. It make installs _more_ predictable, which minimises "but it works on my machine" issues.
-1. It minimises changes to the `package-lock.json` (if your project is comitting that file).
+#### Prefer `npm ci` to `npm install` where possible
 
-A problem with using `npm install` is that it doesn't guarantee reproducible builds, as neither the `package.json` nor `package-lock.json` is a source of authority for what's installed.
+The `npm ci` command was introduced with `npm` version 5.7.0+ ([node v10.3+ or 8.12+](https://nodejs.org/en/download/releases/)). 
```

Comment:
For simplicity, maybe we could remove this altogether. We shouldn't be using a version of Node.js that bundles a version of npm that doesn't support `npm ci`, so removing this would make the document less scary and easier to understand.

Reply:
Agreed